connector/fyle_integrations_platform_connector/apis/base.py

```diff
@@ -62,6 +62,17 @@ def get_all_generator(self):
 
         return self.connection.list_all(query_params)
 
+    def post(self, payload):
+        """
+        Post data to Fyle
+        """
+        return self.connection.post(payload)
+
+    def post_bulk(self, payload):
+        """
+        Post data to Fyle in Bulk
+        """
+        return self.connection.post_bulk(payload)
```

Comment:
let's get a list from clients and add {data: payload} since the format is same for bulk post of every apis